ide/libs.graalsdk.system/nbproject/project.xml

```diff
@@ -82,12 +82,36 @@
             </test-dependencies>
             <public-packages/>
             <class-path-extension>
-                <runtime-relative-path></runtime-relative-path>
-                <binary-origin>external/graal-sdk-20.3.0.jar</binary-origin>
+                <runtime-relative-path>ext/launcher-common-24.0.0.jar</runtime-relative-path>
+                <binary-origin>external/launcher-common-24.0.0.jar</binary-origin>
             </class-path-extension>
             <class-path-extension>
-                <runtime-relative-path></runtime-relative-path>
-                <binary-origin>external/launcher-common-20.3.0.jar</binary-origin>
+                <runtime-relative-path>ext/jline-24.0.0.jar</runtime-relative-path>
+                <binary-origin>external/jline-24.0.0.jar</binary-origin>
+            </class-path-extension>
+            <class-path-extension>
+                <runtime-relative-path>ext/collections-24.0.0.jar</runtime-relative-path>
+                <binary-origin>external/collections-24.0.0.jar</binary-origin>
+            </class-path-extension>
+            <class-path-extension>
+                <runtime-relative-path>ext/jniutils-24.0.0.jar</runtime-relative-path>
+                <binary-origin>external/jniutils-24.0.0.jar</binary-origin>
+            </class-path-extension>
+            <class-path-extension>
+                <runtime-relative-path>ext/launcher-common-24.0.0.jar</runtime-relative-path>
+                <binary-origin>external/launcher-common-24.0.0.jar</binary-origin>
+            </class-path-extension>
+            <class-path-extension>
+                <runtime-relative-path>ext/nativeimage-24.0.0.jar</runtime-relative-path>
+                <binary-origin>external/nativeimage-24.0.0.jar</binary-origin>
+            </class-path-extension>
+            <class-path-extension>
+                <runtime-relative-path>ext/polyglot-24.0.0.jar</runtime-relative-path>
+                <binary-origin>external/polyglot-24.0.0.jar</binary-origin>
+            </class-path-extension>
+            <class-path-extension>
+                <runtime-relative-path>ext/word-24.0.0.jar</runtime-relative-path>
+                <binary-origin>external/word-24.0.0.jar</binary-origin>
```

Comment:
I think, that the `<runtime-relative-path>` entries should not be here/be empty. When running `ant tryme`, I see:

```
WARNING [org.netbeans.core.startup.InstalledFileLocatorImpl]: module org.netbeans.libs.graalsdk.system in /home/matthias/src/netbeans/nbbuild/netbeans/ide does not own modules/ext/launcher-common-24.0.0.jar at org.netbeans.LocaleVariants.findLogicalPath(LocaleVariants.java:246)
WARNING [org.netbeans.core.startup.InstalledFileLocatorImpl]: module org.netbeans.libs.graalsdk.system in /home/matthias/src/netbeans/nbbuild/netbeans/ide does not own modules/ext/jline-24.0.0.jar at org.netbeans.LocaleVariants.findLogicalPath(LocaleVariants.java:246)
WARNING [org.netbeans.core.startup.InstalledFileLocatorImpl]: module org.netbeans.libs.graalsdk.system in /home/matthias/src/netbeans/nbbuild/netbeans/ide does not own modules/ext/collections-24.0.0.jar at org.netbeans.LocaleVariants.findLogicalPath(LocaleVariants.java:246)
WARNING [org.netbeans.core.startup.InstalledFileLocatorImpl]: module org.netbeans.libs.graalsdk.system in /home/matthias/src/netbeans/nbbuild/netbeans/ide does not own modules/ext/jniutils-24.0.0.jar at org.netbeans.LocaleVariants.findLogicalPath(LocaleVariants.java:246)
WARNING [org.netbeans.core.startup.InstalledFileLocatorImpl]: module org.netbeans.libs.graalsdk.system in /home/matthias/src/netbeans/nbbuild/netbeans/ide does not own modules/ext/nativeimage-24.0.0.jar at org.netbeans.LocaleVariants.findLogicalPath(LocaleVariants.java:246)
WARNING [org.netbeans.core.startup.InstalledFileLocatorImpl]: module org.netbeans.libs.graalsdk.system in /home/matthias/src/netbeans/nbbuild/netbeans/ide does not own modules/ext/polyglot-24.0.0.jar at org.netbeans.LocaleVariants.findLogicalPath(LocaleVariants.java:246)
WARNING [org.netbeans.core.startup.InstalledFileLocatorImpl]: module org.netbeans.libs.graalsdk.system in /home/matthias/src/netbeans/nbbuild/netbeans/ide does not own modules/ext/word-24.0.0.jar at org.netbeans.LocaleVariants.findLogicalPath(LocaleVariants.java:246)
WARNING [org.netbeans.core.startup.NbEvents]: The extension /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/ext/launcher-common-24.0.0.jar may be multiply loaded by modules: [/home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk-system.jar, /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk.jar]; see: http://www.netbeans.org/download/dev/javadoc/org-openide-modules/org/openide/modules/doc-files/classpath.html#class-path
WARNING [org.netbeans.core.startup.NbEvents]: The extension /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/ext/jline-24.0.0.jar may be multiply loaded by modules: [/home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk-system.jar, /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk.jar]; see: http://www.netbeans.org/download/dev/javadoc/org-openide-modules/org/openide/modules/doc-files/classpath.html#class-path
WARNING [org.netbeans.core.startup.NbEvents]: The extension /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/ext/collections-24.0.0.jar may be multiply loaded by modules: [/home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk-system.jar, /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk.jar]; see: http://www.netbeans.org/download/dev/javadoc/org-openide-modules/org/openide/modules/doc-files/classpath.html#class-path
WARNING [org.netbeans.core.startup.NbEvents]: The extension /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/ext/jniutils-24.0.0.jar may be multiply loaded by modules: [/home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk-system.jar, /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk.jar]; see: http://www.netbeans.org/download/dev/javadoc/org-openide-modules/org/openide/modules/doc-files/classpath.html#class-path
WARNING [org.netbeans.core.startup.NbEvents]: The extension /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/ext/nativeimage-24.0.0.jar may be multiply loaded by modules: [/home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk-system.jar, /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk.jar]; see: http://www.netbeans.org/download/dev/javadoc/org-openide-modules/org/openide/modules/doc-files/classpath.html#class-path
WARNING [org.netbeans.core.startup.NbEvents]: The extension /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/ext/polyglot-24.0.0.jar may be multiply loaded by modules: [/home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk-system.jar, /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk.jar]; see: http://www.netbeans.org/download/dev/javadoc/org-openide-modules/org/openide/modules/doc-files/classpath.html#class-path
WARNING [org.netbeans.core.startup.NbEvents]: The extension /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/ext/word-24.0.0.jar may be multiply loaded by modules: [/home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk-system.jar, /home/matthias/src/netbeans/nbbuild/netbeans/ide/modules/org-netbeans-libs-graalsdk.jar]; see: http://www.netbeans.org/download/dev/javadoc/org-openide-modules/org/openide/modules/doc-files/classpath.html#class-path
```

I understood the original version in the way, that the `binary-origin` is used at development time and the `runtime-relative-path` is empty, so that at runtime the binaries are not visibile to the module system (at least not via `graalsdk.system`.

Reply:
>the references to the GraalVM implementations at runtime in libs.graalsdk.system make no sense to me.

Thanks for noticing ! Seem I was copying from libs.graal to libs.graal.system (for compilation purposes) and did not prune after.  Addressed in b43e68dbdd